notebooks/tel_and_site/subsys_req_ver/tma/SITCOMTN-067_Vel_Accel_Jerk_15Nov23.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    print(len(az_track), len(el_track), len(azPos), len(elPos))
</pre><p>Add strings describing the variables printed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Done